django_prices/widgets.py

```diff
@@ -1,27 +1,20 @@
 from django import forms
-from django.template.loader import render_to_string
 from prices import Money
 
 __all__ = ["MoneyInput"]
 
 
-class MoneyInput(forms.TextInput):
-    template = "prices/widget.html"
-    input_type = "number"
+class MoneyInput(forms.MultiWidget):
+    def __init__(self, attrs=None):
+        widgets = [
+            forms.TextInput(attrs={"type": "number", "step": "any"}),
+            forms.TextInput(),
```

Comment:
I don't think that's usable in any real-life situation. You'd pretty much always want to limit the choices here (and probably replace the selection if only one option is given).